modules/lambda-registrator/variables.tf

```diff
@@ -123,7 +123,7 @@ variable "consul_lambda_registrator_image" {
   default     = "public.ecr.aws/hashicorp/consul-lambda-registrator:0.1.0-beta4"
 
   validation {
-    condition     = can(regex("^[a-zA-Z0-9_.-]+/[a-z0-9_.-]+/[a-z0-9_.-]+:[a-zA-Z0-9_.-]+$", var.consul_lambda_registrator_image))
+    condition     = can(regex(".+?/.+?:.+", var.consul_lambda_registrator_image))
```

Comment:
What is the reason for relaxing this validation? Even before, it doesn't look like this was quite right since it requires a tag, but not entering a tag is still valid and defaults to `latest`.